src/screens/Survey/index.tsx

```diff
@@ -1,48 +1,90 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { ToastContainer, toast } from 'react-toastify';
 
 import { ReactComponent as ArrowBack } from 'assets/images/icons/arrow-back.svg';
 import ElevatedButton from 'components/ElevatedButton';
+import LoadingDialog from 'components/LoadingDialog';
 import MainView from 'components/MainView';
+import { useAppDispatch, useAppSelector } from 'hooks';
+import { getSurveyAsyncThunk } from 'store/reducers/Survey';
+
+import 'react-toastify/dist/ReactToastify.css';
 
 export const surveyScreenTestIds = {
   backButton: 'survey__back-button',
   coverImage: 'survey__cover-image',
   title: 'survey__title-text',
   description: 'survey__description-text',
   startSurveyButton: 'survey__start-survey-button',
+  loadingDialog: 'survey__loading-dialog',
+  toast: 'survey__toast',
 };
 
 const SurveyScreen = (): JSX.Element => {
+  const { id } = useParams();
+
+  const { survey, isLoading, isError } = useAppSelector((state) => state.survey);
+
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+
+  function goBack() {
+    navigate(-1);
+  }
+
+  useEffect(() => {
+    dispatch(getSurveyAsyncThunk(id ?? ''));
+  }, [dispatch, id]);
+
+  useEffect(() => {
+    if (isError) {
+      toast.error('There is something wrong. Please try again later!', { position: 'top-center' });
+    }
+  }, [isError]);
+
   return (
-    <MainView backgroundUrl="https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_">
-      <div className="flex flex-col">
-        <button type="button" className="w-fit mt-[37px] ml-[39px] p-1 text-white" data-test-id={surveyScreenTestIds.backButton}>
-          <ArrowBack />
-        </button>
-        <div className="w-1/2 self-center pt-36">
-          <img
-            src="https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l"
-            className="w-full h-[302px] rounded-[12px] object-cover"
-            alt="survey"
-            data-test-id={surveyScreenTestIds.coverImage}
-          />
-          <p className="text-white text-x-large font-extrabold pt-8" data-test-id={surveyScreenTestIds.title}>
-            Working from home Check-In
-          </p>
-          <p
-            className="text-white text-regular tracking-survey-tight opacity-60 pt-2"
-            data-test-id={surveyScreenTestIds.description}
-          >
-            We would like to know how you feel about our work from home (WFH) experience.
-          </p>
-          <div className="pt-8">
-            <ElevatedButton isFullWidth type="submit" data-test-id={surveyScreenTestIds.startSurveyButton}>
-              Start Survey
-            </ElevatedButton>
+    <div>
+      <ToastContainer />
```

Comment:
I think it should be placed in the App file for global toasting 🤔 Unless you will place every file needed to toast the message.

Reply:
@nvminhtue nice suggestion 🚀 Thanks